spec/requests/api/job_executions_spec.rb

```diff
@@ -115,6 +115,7 @@
         job:     job,
         queue:   job_queue.name,
         message: ActionController::Parameters.new(message),
+        delay_seconds: nil,
```

Comment:
[nits] Does it need?

Reply:
Yes, needed. Barbeque::Api::JobExecutionsController always calls Barbeque::MessageEnqueuingService.new  with delay_seconds parameter (while it could be nil).